src/Primrose/BrowserEnvironment.js

```diff
@@ -366,16 +367,11 @@ export default class BrowserEnvironment extends EventDispatcher {
     };
 
     const doPicking = () => {
-      updateAll();
+      updateAll();  
       this.input.resolvePicking(this.scene);
     };
 
-    const moveGround = () => {
```

Comment:
Is this something you're still interested in doing? I can take care of it if you're too busy.